python/pyfunc-server/pyfuncserver/protocol/rest/server_sanic.py

```diff
@@ -0,0 +1,93 @@
+import json
+import logging
+
+from merlin.pyfunc import PyFuncOutput
+from orjson import dumps, loads
+from prometheus_client import CollectorRegistry
+from prometheus_client.exposition import choose_encoder
+from sanic import Sanic
+from sanic import json as sanic_json
+from sanic import raw, text
+from sanic.request import Request
+
+from pyfuncserver.config import Config
+from pyfuncserver.model.model import PyFuncModel
+
+
+class HTTPServer:
+    def __init__(
+        self, config: Config, metrics_registry: CollectorRegistry, model: PyFuncModel
+    ):
+
+        self.config = config
+        self.workers = config.workers
+        self.model_manifest = config.model_manifest
+        self.http_port = config.http_port
+        self.metrics_registry = metrics_registry
+        self.registered_models: dict = {}
+        self.register_model(model)
+
+        Sanic.start_method = "fork"
+        self.app = Sanic("pyfunc")
+        self.app.update_config({"ACCESS_LOG": False})
```

Comment:
turn access log off for better performance - ref https://sanic.dev/en/guide/running/running.html#performance-considerations